src/coreclr/jit/emitarm64sve.cpp

```diff
@@ -3774,18 +3774,22 @@ void emitter::emitInsSve_R_R_R(instruction     ins,
                 // ORR is an alias for MOV, and is always the preferred disassembly.
                 ins = INS_sve_orr;
             }
-            else if (isPredicateRegister(reg3) &&
+            // TODO-SVE: Remove check for (opt == INS_OPTS_SCALABLE_B) when predicate registers
+            //  are present.
+            else if ((opt == INS_OPTS_SCALABLE_B) && isPredicateRegister(reg3) &&
```

Comment:
You are using `INS_OPTS_SCALABLE_B` to check that you have a predicate? What about if moving a byte vector? Maybe add another enum value instead?

Reply:
yes, I don't need this check anymore after I added `INS_SCALABLE_OPTS_PREDICATE_MERGE_MOV`